concordium-sdk/src/main/java/com/concordium/sdk/responses/poolstatus/PendingChangeRemovePool.java

```diff
@@ -0,0 +1,18 @@
+package com.concordium.sdk.responses.poolstatus;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Date;
+
+@Getter
+@ToString
+@Data
+public class PendingChangeRemovePool extends PendingChange {
+
+    /**
+     * Effective time of the change.
+     */
+    private Date effectiveTime;
```

Comment:
Please make final. 
And I don't see the need for `@ToString` and `@Getter` when `@Data` is being used.

Reply:
done